packages/contracts/src/core/dao/DAO.sol

```diff
@@ -122,6 +154,28 @@ contract DAO is
         __PermissionManager_init(_initialOwner);
     }
 
+    /// @notice Initializes the DAO after an upgrade from a previous version.
+    /// @param _protocolVersion The protocol version number of the previous DAO implementation contract this upgrade is transitioning from.
+    /// @param _initData The initialization data to be passed to via `upgradeToAndCall` (see [ERC-1967](https://docs.openzeppelin.com/contracts/4.x/api/proxy#ERC1967Upgrade)).
+    function initializeFrom(
+        uint8[3] calldata _protocolVersion,
```

Comment:
```suggestion
        uint8[3] calldata _oldVersion,
```

Reply:
`previousProtocolVersion`? I think it is important to explicitly write that this is about the OSX protocol version in contrast to just `previousVersion` because this could be misinterpreted by some version number specific to the contract.

I thought `initializeFrom(protocolVersion)` would already make clear that this is some older/previous version, but I am happy to make this longer by writing `initializeFrom(previousProtocolVersion)` for the sake of being explicit.

Reply:
Changed in a97a9bac9e2bc56663a80089abe2562caebcfced